### ISSUE TITLE
4K MONITORS AND RS

### ISSUE THREAD

**Issue by felisucoibi** [RetroShare/RetroShare#1330]
Does anybody tried or use rs in 4k? i don't know how to change fots, etc, the only thing i was able is to change icons to 128x128 that btw has a bug with home icon. But everything is supersmall.

**Comment by felisucoibi** [comment]
ups, it just selffixed retrartin RS, didnt know this was possible. It changed fonts etc to use the new gnome default ones.... btw the bug of HOME ICON is there. Is smaller in 4k THAN THE REST...

**Comment by G10h4ck** [comment]
I have eared from a friend on a retina like display (very high resolution but not mac) that he saw RetroShare icons supersmall and almost invisible in many cases but he could not find any way to fix it, we should probably switch to SVG icons at some point...

**Comment by felisucoibi** [comment]
True, icons keep small, the main icons can be resized manually to 128x128, but the rest of icons are small, they already where small in 2k resolution, like the forum ones.... the icons are smaller than the mouse cursor :P
